### ISSUE TITLE
Analyze diffie-hellman 5.0.3

### ISSUE THREAD

**Issue by balhar-jakub** [zowe/zlux#1011]

**Comment by 1000TurquoisePogs** [comment]
we use this library because of a performance regression on node long ago.
i would like to retest this to see how native vs library compares today to see if it can be removed.

**Comment by 1000TurquoisePogs** [comment]
using `crypto` in node is still extraordinarily slow, which was the reason we used this `diffie-hellman` instead. 
however, research revealed `crypto.webcrypto` has functions that are implemented differently that may be faster than both. so, it'd be great to move to that, but unfortunately the API is organized differently so our code would need to change for SSH.

Rather than improve our ssh parser i'd rather move to a different ssh library, which has been on the to-do list forever. ( https://github.com/zowe/zlux/issues/427 ) ( https://www.npmjs.com/package/ssh2 )

Basically, there's nothing to do with this library now, but I would like to find a better solution long-term as it was never desired to begin with.
